projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/vdk-jupyterlab-extension/vdk_ui.py

```diff
@@ -45,53 +47,58 @@ def run_job(path, arguments=None):
             return f"{process.returncode}"
 
     @staticmethod
-    def delete_job(name: str, team: str, rest_api_url: str):
+    def delete_job(job_data: DictObj):
         """
         Execute `delete job`.
-        :param name: the name of the data job that will be deleted
-        :param team: the team of the data job that will be deleted
-        :param rest_api_url: The base REST API URL.
+        :param job_data: the job data
+        following the projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/dataClasses/jobDataModel.json
         :return: message that the job is deleted
         """
-        cmd = JobDelete(rest_api_url)
-        cmd.delete_job(name, team)
-        return f"Deleted the job with name {name} from {team} team. "
+        cmd = JobDelete(job_data.restApiUrl)
+        cmd.delete_job(job_data.jobName, job_data.jobTeam)
```

Comment:
Is the contract on job_data ensuring a `jobName` property presence?

Reply:
yes, the job_data object follows the projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/dataClasses/jobDataModel.json